src/COBREXA.jl

```diff
@@ -32,6 +32,7 @@ _inc_all.(
             joinpath("base", "logging"),
             joinpath("base", "macros"),
             joinpath("base", "types"),
+            joinpath("base", "types", "derivedmodels"),
```

Comment:
ah yes this is good idea to solve many problems :]

Perhaps we should have literal `wrappers/`